tests/test_models.py

```diff
@@ -78,3 +79,39 @@ def test_subset_balance_doesnt_fail_superset(self,):
         Ledger.transactions.
         """
         assert self.ledger.subset_balance(Transaction.objects.all()) == self.ledger.balance()
+
+    def test_acquire_lock_release_lock(self):
+        """
+        Create one hypothetical sequence consisting of three actors and two ledgers.  Each Ledger should only allow one
+        lock to be grabbed at a time.
+        """
+        # Simple case, acquire lock on first ledger.
+        lock_1 = self.ledger.acquire_lock()
+        assert lock_1
+        # A second actor attempts to acquire lock on first ledger, but it's already locked.
+        lock_2 = self.ledger.acquire_lock()
+        assert lock_2 is None
+        # A third actor attempts to acquire lock on second ledger, should work even though first ledger is locked.
+        lock_3 = self.other_ledger.acquire_lock()
```

Comment:
You might want to release this lock, or add some generic cleanup to release all locks.  I know this shouldn't be a concern right now, because we're creating `other_ledger` via `setUp` before each test function, but if we ever convert `setUp` to `setUpClass` or anything like that, it'd be easy for us to get confused about why this lock is never acquirable from other tests.